### ISSUE TITLE
ci: Tilt terra tests are failing intermittently

### ISSUE THREAD

**Issue by tbjump** [wormhole-foundation/wormhole#3106]
Tilt run where only the terra wormhole-sdk tests failed: https://github.com/wormhole-foundation/wormhole/actions/runs/5304176933/attempts/1
Re-run where they succeeded: https://github.com/wormhole-foundation/wormhole/actions/runs/5304176933/attempts/2

**Comment by tbjump** [comment]
```
2023-06-18T14:33:16.0706018Z  sdk-ci-tests │ FAIL src/token_bridge/__tests__/terra-integration.ts (95.335 s)
2023-06-18T14:33:16.0706718Z  sdk-ci-tests │   Terra Integration Tests
2023-06-18T14:33:16.0707577Z  sdk-ci-tests │     Terra deposit and transfer tokens
2023-06-18T14:33:16.0708590Z  sdk-ci-tests │       ✓ Tokens transferred can't exceed tokens deposited (3411 ms)
2023-06-18T14:33:16.0709506Z  sdk-ci-tests │     Ethereum to Terra
2023-06-18T14:33:16.0710666Z  sdk-ci-tests │       ✕ Attest Ethereum ERC-20 to Terra (2791 ms)
2023-06-18T14:33:16.0711568Z  sdk-ci-tests │       ✓ Ethereum ERC-20 is attested on Terra (3 ms)
2023-06-18T14:33:16.0712094Z  sdk-ci-tests │       ✕ Send Ethereum ERC-20 to Terra (312 ms)
2023-06-18T14:33:16.0712445Z  sdk-ci-tests │     Terra to Ethereum
2023-06-18T14:33:16.0713133Z  sdk-ci-tests │       ✓ Attestation from Terra to ETH (6084 ms)
2023-06-18T14:33:16.0713542Z  sdk-ci-tests │       ✓ Transfer from Terra (9147 ms)
2023-06-18T14:33:16.0714044Z  sdk-ci-tests │       ✓ Transfer wrapped luna back to Terra (7349 ms)
2023-06-18T14:33:16.0714922Z  sdk-ci-tests │     Terra <=> Ethereum roundtrip
2023-06-18T14:33:16.0715817Z  sdk-ci-tests │       ✓ Transfer CW20 token from Terra to Ethereum and back again (14491 ms)
2023-06-18T14:33:16.0716184Z  sdk-ci-tests │
2023-06-18T14:33:16.0716627Z  sdk-ci-tests │   ● Terra Integration Tests › Ethereum to Terra › Attest Ethereum ERC-20 to Terra
2023-06-18T14:33:16.0716980Z  sdk-ci-tests │
2023-06-18T14:33:16.0717407Z  sdk-ci-tests │     Failed: "An error occurred while trying to attest from Ethereum to Terra"
2023-06-18T14:33:16.0717751Z  sdk-ci-tests │
2023-06-18T14:33:16.0718043Z  sdk-ci-tests │       273 |         } catch (e) {
2023-06-18T14:33:16.0718403Z  sdk-ci-tests │       274 |           console.error(e);
2023-06-18T14:33:16.0718761Z  sdk-ci-tests │     > 275 |           done(
2023-06-18T14:33:16.0719059Z  sdk-ci-tests │           |           ^
2023-06-18T14:33:16.0719505Z  sdk-ci-tests │       276 |             "An error occurred while trying to attest from Ethereum to Terra"
2023-06-18T14:33:16.0719887Z  sdk-ci-tests │       277 |           );
2023-06-18T14:33:16.0720193Z  sdk-ci-tests │       278 |         }
2023-06-18T14:33:16.0732370Z  sdk-ci-tests │
2023-06-18T14:33:16.0733680Z  sdk-ci-tests │       at src/token_bridge/__tests__/terra-integration.ts:275:11
2023-06-18T14:33:16.0734708Z  sdk-ci-tests │           at Generator.throw (<anonymous>)
2023-06-18T14:33:16.0735783Z  sdk-ci-tests │       at rejected (src/token_bridge/__tests__/terra-integration.ts:6:65)
2023-06-18T14:33:16.0736372Z  sdk-ci-tests │
2023-06-18T14:33:16.0737196Z  sdk-ci-tests │   ● Terra Integration Tests › Ethereum to Terra › Send Ethereum ERC-20 to Terra
2023-06-18T14:33:16.0737566Z  sdk-ci-tests │
2023-06-18T14:33:16.0737963Z  sdk-ci-tests │     Failed: "An error occurred while trying to send from Ethereum to Terra"
2023-06-18T14:33:16.0738336Z  sdk-ci-tests │
2023-06-18T14:33:16.0738950Z  sdk-ci-tests │       464 |         } catch (e) {
2023-06-18T14:33:16.0739311Z  sdk-ci-tests │       465 |           console.error(e);
2023-06-18T14:33:16.0739806Z  sdk-ci-tests │     > 466 |           done("An error occurred while trying to send from Ethereum to Terra");
2023-06-18T14:33:16.0740191Z  sdk-ci-tests │           |           ^
2023-06-18T14:33:16.0740846Z  sdk-ci-tests │       467 |         }
2023-06-18T14:33:16.0741181Z  sdk-ci-tests │       468 |       })();
2023-06-18T14:33:16.0741506Z  sdk-ci-tests │       469 |     });
2023-06-18T14:33:16.0741896Z  sdk-ci-tests │
2023-06-18T14:33:16.0742327Z  sdk-ci-tests │       at src/token_bridge/__tests__/terra-integration.ts:466:11
2023-06-18T14:33:16.0743108Z  sdk-ci-tests │       at fulfilled (src/token_bridge/__tests__/terra-integration.ts:5:58)
```

**Comment by evan-gray** [comment]
This occurred before: 
```
2023-06-18T14:32:38.6414806Z  sdk-ci-tests │   console.error
2023-06-18T14:32:38.6416095Z  sdk-ci-tests │     Error: socket hang up
2023-06-18T14:32:38.6417143Z  sdk-ci-tests │         at connResetException (node:internal/errors:717:14)
2023-06-18T14:32:38.6418134Z  sdk-ci-tests │         at Socket.socketOnEnd (node:_http_client:520:23)
2023-06-18T14:32:38.6418711Z  sdk-ci-tests │         at Socket.emit (node:events:524:35)
2023-06-18T14:32:38.6419190Z  sdk-ci-tests │         at endReadableNT (node:internal/streams/readable:1359:12)
2023-06-18T14:32:38.6419769Z  sdk-ci-tests │         at processTicksAndRejections (node:internal/process/task_queues:82:21) {
2023-06-18T14:32:38.6420211Z  sdk-ci-tests │       code: 'ECONNRESET',
2023-06-18T14:32:38.6420528Z  sdk-ci-tests │       config: {
2023-06-18T14:32:38.6420852Z  sdk-ci-tests │         transitional: {
2023-06-18T14:32:38.6421230Z  sdk-ci-tests │           silentJSONParsing: true,
2023-06-18T14:32:38.6421608Z  sdk-ci-tests │           forcedJSONParsing: true,
2023-06-18T14:32:38.6422023Z  sdk-ci-tests │           clarifyTimeoutError: false
2023-06-18T14:32:38.6422384Z  sdk-ci-tests │         },
2023-06-18T14:32:38.6423262Z  sdk-ci-tests │         adapter: [Function: httpAdapter],
2023-06-18T14:32:38.6424267Z  sdk-ci-tests │         transformRequest: [ [Function: transformRequest] ],
2023-06-18T14:32:38.6424793Z  sdk-ci-tests │         transformResponse: [ [Function: transformResponse] ],
2023-06-18T14:32:38.6425194Z  sdk-ci-tests │         timeout: 0,
2023-06-18T14:32:38.6425554Z  sdk-ci-tests │         xsrfCookieName: 'XSRF-TOKEN',
2023-06-18T14:32:38.6425962Z  sdk-ci-tests │         xsrfHeaderName: 'X-XSRF-TOKEN',
2023-06-18T14:32:38.6426347Z  sdk-ci-tests │         maxContentLength: -1,
2023-06-18T14:32:38.6426692Z  sdk-ci-tests │         maxBodyLength: -1,
2023-06-18T14:32:38.6427123Z  sdk-ci-tests │         validateStatus: [Function: validateStatus],
2023-06-18T14:32:38.6427497Z  sdk-ci-tests │         headers: {
2023-06-18T14:32:38.6427890Z  sdk-ci-tests │           Accept: 'application/json, text/plain, */*',
2023-06-18T14:32:38.6428331Z  sdk-ci-tests │           'User-Agent': 'axios/0.24.0'
2023-06-18T14:32:38.6428718Z  sdk-ci-tests │         },
2023-06-18T14:32:38.6429035Z  sdk-ci-tests │         method: 'get',
2023-06-18T14:32:38.6429483Z  sdk-ci-tests │         url: 'http://terra-fcd:3060/v1/txs/gas_prices',
2023-06-18T14:32:38.6429866Z  sdk-ci-tests │         data: undefined
2023-06-18T14:32:38.6430170Z  sdk-ci-tests │       },
2023-06-18T14:32:38.6431531Z  sdk-ci-tests │       request: <ref *1> Writable {
2023-06-18T14:32:38.6431932Z  sdk-ci-tests │         _writableState: WritableState {
2023-06-18T14:32:38.6432480Z  sdk-ci-tests │           objectMode: false,
2023-06-18T14:32:38.6433270Z  sdk-ci-tests │           highWaterMark: 16384,
2023-06-18T14:32:38.6433990Z  sdk-ci-tests │           finalCalled: false,
2023-06-18T14:32:38.6434737Z  sdk-ci-tests │           needDrain: false,
2023-06-18T14:32:38.6435301Z  sdk-ci-tests │           ending: false,
2023-06-18T14:32:38.6449379Z  sdk-ci-tests │           ended: false,
2023-06-18T14:32:38.6450195Z  sdk-ci-tests │           finished: false,
2023-06-18T14:32:38.6450857Z  sdk-ci-tests │           destroyed: false,
2023-06-18T14:32:38.6452014Z  sdk-ci-tests │           decodeStrings: true,
2023-06-18T14:32:38.6452957Z  sdk-ci-tests │           defaultEncoding: 'utf8',
2023-06-18T14:32:38.6453438Z  sdk-ci-tests │           length: 0,
2023-06-18T14:32:38.6453988Z  sdk-ci-tests │           writing: false,
2023-06-18T14:32:38.6454683Z  sdk-ci-tests │           corked: 0,
2023-06-18T14:32:38.6455299Z  sdk-ci-tests │           sync: true,
2023-06-18T14:32:38.6455866Z  sdk-ci-tests │           bufferProcessing: false,
2023-06-18T14:32:38.6456483Z  sdk-ci-tests │           onwrite: [Function: bound onwrite],
2023-06-18T14:32:38.6457193Z  sdk-ci-tests │           writecb: null,
2023-06-18T14:32:38.6457682Z  sdk-ci-tests │           writelen: 0,
2023-06-18T14:32:38.6458381Z  sdk-ci-tests │           afterWriteTickInfo: null,
2023-06-18T14:32:38.6459062Z  sdk-ci-tests │           buffered: [],
2023-06-18T14:32:38.6460032Z  sdk-ci-tests │           bufferedIndex: 0,
2023-06-18T14:32:38.6460915Z  sdk-ci-tests │           allBuffers: true,
2023-06-18T14:32:38.6461516Z  sdk-ci-tests │           allNoop: true,
2023-06-18T14:32:38.6462083Z  sdk-ci-tests │           pendingcb: 0,
2023-06-18T14:32:38.6462933Z  sdk-ci-tests │           constructed: true,
2023-06-18T14:32:38.6463669Z  sdk-ci-tests │           prefinished: false,
2023-06-18T14:32:38.6464453Z  sdk-ci-tests │           errorEmitted: false,
2023-06-18T14:32:38.6465169Z  sdk-ci-tests │           emitClose: true,
2023-06-18T14:32:38.6465828Z  sdk-ci-tests │           autoDestroy: true,
2023-06-18T14:32:38.6466541Z  sdk-ci-tests │           errored: null,
2023-06-18T14:32:38.6467252Z  sdk-ci-tests │           closed: false,
2023-06-18T14:32:38.6467984Z  sdk-ci-tests │           closeEmitted: false,
2023-06-18T14:32:38.6468856Z  sdk-ci-tests │           [Symbol(kOnFinished)]: []
2023-06-18T14:32:38.6469548Z  sdk-ci-tests │         },
2023-06-18T14:32:38.6470311Z  sdk-ci-tests │         _events: [Object: null prototype] {
2023-06-18T14:32:38.6471257Z  sdk-ci-tests │           response: [Function: handleResponse],
2023-06-18T14:32:38.6472132Z  sdk-ci-tests │           error: [Function: handleRequestError]
2023-06-18T14:32:38.6473049Z  sdk-ci-tests │         },
2023-06-18T14:32:38.6473554Z  sdk-ci-tests │         _eventsCount: 2,
2023-06-18T14:32:38.6473940Z  sdk-ci-tests │         _maxListeners: undefined,
2023-06-18T14:32:38.6474294Z  sdk-ci-tests │         _options: {
2023-06-18T14:32:38.6474649Z  sdk-ci-tests │           maxRedirects: 21,
2023-06-18T14:32:38.6475030Z  sdk-ci-tests │           maxBodyLength: 10485760,
2023-06-18T14:32:38.6475455Z  sdk-ci-tests │           protocol: 'http:',
2023-06-18T14:32:38.6475890Z  sdk-ci-tests │           path: '/v1/txs/gas_prices',
2023-06-18T14:32:38.6476732Z  sdk-ci-tests │           method: 'GET',
2023-06-18T14:32:38.6477271Z  sdk-ci-tests │           headers: [Object],
2023-06-18T14:32:38.6477810Z  sdk-ci-tests │           agent: undefined,
2023-06-18T14:32:38.6478521Z  sdk-ci-tests │           agents: [Object],
2023-06-18T14:32:38.6479347Z  sdk-ci-tests │           auth: undefined,
2023-06-18T14:32:38.6480162Z  sdk-ci-tests │           hostname: 'terra-fcd',
2023-06-18T14:32:38.6481262Z  sdk-ci-tests │           port: '3060',
2023-06-18T14:32:38.6482444Z  sdk-ci-tests │           nativeProtocols: [Object],
2023-06-18T14:32:38.6483745Z  sdk-ci-tests │           pathname: '/v1/txs/gas_prices'
2023-06-18T14:32:38.6484808Z  sdk-ci-tests │         },
2023-06-18T14:32:38.6485841Z  sdk-ci-tests │         _ended: true,
2023-06-18T14:32:38.6486891Z  sdk-ci-tests │         _ending: true,
2023-06-18T14:32:38.6488003Z  sdk-ci-tests │         _redirectCount: 0,
2023-06-18T14:32:38.6489189Z  sdk-ci-tests │         _redirects: [],
2023-06-18T14:32:38.6490233Z  sdk-ci-tests │         _requestBodyLength: 0,
2023-06-18T14:32:38.6491331Z  sdk-ci-tests │         _requestBodyBuffers: [],
2023-06-18T14:32:38.6493101Z  sdk-ci-tests │         _onNativeResponse: [Function (anonymous)],
2023-06-18T14:32:38.6494450Z  sdk-ci-tests │         _currentRequest: ClientRequest {
2023-06-18T14:32:38.6496196Z  sdk-ci-tests │           _events: [Object: null prototype],
2023-06-18T14:32:38.6497379Z  sdk-ci-tests │           _eventsCount: 7,
2023-06-18T14:32:38.6498650Z  sdk-ci-tests │           _maxListeners: undefined,
2023-06-18T14:32:38.6499662Z  sdk-ci-tests │           outputData: [],
2023-06-18T14:32:38.6500253Z  sdk-ci-tests │           outputSize: 0,
2023-06-18T14:32:38.6500879Z  sdk-ci-tests │           writable: true,
2023-06-18T14:32:38.6502139Z  sdk-ci-tests │           destroyed: false,
2023-06-18T14:32:38.6503373Z  sdk-ci-tests │           _last: true,
2023-06-18T14:32:38.6504515Z  sdk-ci-tests │           chunkedEncoding: false,
2023-06-18T14:32:38.6505748Z  sdk-ci-tests │           shouldKeepAlive: true,
2023-06-18T14:32:38.6507164Z  sdk-ci-tests │           maxRequestsOnConnectionReached: false,
2023-06-18T14:32:38.6508984Z  sdk-ci-tests │           _defaultKeepAlive: true,
2023-06-18T14:32:38.6510423Z  sdk-ci-tests │           useChunkedEncodingByDefault: false,
2023-06-18T14:32:38.6511727Z  sdk-ci-tests │           sendDate: false,
2023-06-18T14:32:38.6513068Z  sdk-ci-tests │           _removedConnection: false,
2023-06-18T14:32:38.6514268Z  sdk-ci-tests │           _removedContLen: false,
2023-06-18T14:32:38.6515468Z  sdk-ci-tests │           _removedTE: false,
2023-06-18T14:32:38.6516714Z  sdk-ci-tests │           strictContentLength: false,
2023-06-18T14:32:38.6517960Z  sdk-ci-tests │           _contentLength: 0,
2023-06-18T14:32:38.6519187Z  sdk-ci-tests │           _hasBody: true,
2023-06-18T14:32:38.6520288Z  sdk-ci-tests │           _trailer: '',
2023-06-18T14:32:38.6521429Z  sdk-ci-tests │           finished: true,
2023-06-18T14:32:38.6522740Z  sdk-ci-tests │           _headerSent: true,
2023-06-18T14:32:38.6523829Z  sdk-ci-tests │           _closed: false,
2023-06-18T14:32:38.6524972Z  sdk-ci-tests │           socket: [Socket],
2023-06-18T14:32:38.6526388Z  sdk-ci-tests │           _header: 'GET /v1/txs/gas_prices HTTP/1.1\r\n' +
2023-06-18T14:32:38.6527934Z  sdk-ci-tests │             'Accept: application/json, text/plain, */*\r\n' +
2023-06-18T14:32:38.6529396Z  sdk-ci-tests │             'User-Agent: axios/0.24.0\r\n' +
2023-06-18T14:32:38.6530691Z  sdk-ci-tests │             'Host: terra-fcd:3060\r\n' +
2023-06-18T14:32:38.6531996Z  sdk-ci-tests │             'Connection: keep-alive\r\n' +
2023-06-18T14:32:38.6533276Z  sdk-ci-tests │             '\r\n',
2023-06-18T14:32:38.6534352Z  sdk-ci-tests │           _keepAliveTimeout: 0,
2023-06-18T14:32:38.6535669Z  sdk-ci-tests │           _onPendingData: [Function: nop],
2023-06-18T14:32:38.6536710Z  sdk-ci-tests │           agent: [Agent],
2023-06-18T14:32:38.6537408Z  sdk-ci-tests │           socketPath: undefined,
2023-06-18T14:32:38.6538063Z  sdk-ci-tests │           method: 'GET',
2023-06-18T14:32:38.6538678Z  sdk-ci-tests │           maxHeaderSize: undefined,
2023-06-18T14:32:38.6539132Z  sdk-ci-tests │           insecureHTTPParser: undefined,
2023-06-18T14:32:38.6539571Z  sdk-ci-tests │           joinDuplicateHeaders: undefined,
2023-06-18T14:32:38.6540009Z  sdk-ci-tests │           path: '/v1/txs/gas_prices',
2023-06-18T14:32:38.6540357Z  sdk-ci-tests │           _ended: false,
2023-06-18T14:32:38.6540694Z  sdk-ci-tests │           res: null,
2023-06-18T14:32:38.6541035Z  sdk-ci-tests │           aborted: false,
2023-06-18T14:32:38.6541453Z  sdk-ci-tests │           timeoutCb: [Function: emitRequestTimeout],
2023-06-18T14:32:38.6542793Z  sdk-ci-tests │           upgradeOrConnect: false,
2023-06-18T14:32:38.6543943Z  sdk-ci-tests │           parser: null,
2023-06-18T14:32:38.6545136Z  sdk-ci-tests │           maxHeadersCount: null,
2023-06-18T14:32:38.6546357Z  sdk-ci-tests │           reusedSocket: true,
2023-06-18T14:32:38.6547537Z  sdk-ci-tests │           host: 'terra-fcd',
2023-06-18T14:32:38.6548759Z  sdk-ci-tests │           protocol: 'http:',
2023-06-18T14:32:38.6550404Z  sdk-ci-tests │           _redirectable: [Circular *1],
2023-06-18T14:32:38.6551709Z  sdk-ci-tests │           [Symbol(kCapture)]: false,
2023-06-18T14:32:38.6553360Z  sdk-ci-tests │           [Symbol(kBytesWritten)]: 0,
2023-06-18T14:32:38.6554594Z  sdk-ci-tests │           [Symbol(kEndCalled)]: true,
2023-06-18T14:32:38.6555923Z  sdk-ci-tests │           [Symbol(kNeedDrain)]: false,
2023-06-18T14:32:38.6557150Z  sdk-ci-tests │           [Symbol(corked)]: 0,
2023-06-18T14:32:38.6558604Z  sdk-ci-tests │           [Symbol(kOutHeaders)]: [Object: null prototype],
2023-06-18T14:32:38.6559944Z  sdk-ci-tests │           [Symbol(errored)]: null,
2023-06-18T14:32:38.6561235Z  sdk-ci-tests │           [Symbol(kUniqueHeaders)]: null
2023-06-18T14:32:38.6562325Z  sdk-ci-tests │         },
2023-06-18T14:32:38.6564027Z  sdk-ci-tests │         _currentUrl: 'http://terra-fcd:3060/v1/txs/gas_prices',
2023-06-18T14:32:38.6565413Z  sdk-ci-tests │         [Symbol(kCapture)]: false
2023-06-18T14:32:38.6566468Z  sdk-ci-tests │       },
2023-06-18T14:32:38.6567623Z  sdk-ci-tests │       response: undefined,
2023-06-18T14:32:38.6568883Z  sdk-ci-tests │       isAxiosError: true,
2023-06-18T14:32:38.6570198Z  sdk-ci-tests │       toJSON: [Function: toJSON]
2023-06-18T14:32:38.6571315Z  sdk-ci-tests │     }
2023-06-18T14:32:38.6572386Z  sdk-ci-tests │ 
2023-06-18T14:32:38.6574091Z  sdk-ci-tests │       272 |           done();
2023-06-18T14:32:38.6575304Z  sdk-ci-tests │       273 |         } catch (e) {
2023-06-18T14:32:38.6576741Z  sdk-ci-tests │     > 274 |           console.error(e);
2023-06-18T14:32:38.6577894Z  sdk-ci-tests │           |                   ^
2023-06-18T14:32:38.6578313Z  sdk-ci-tests │       275 |           done(
2023-06-18T14:32:38.6579090Z  sdk-ci-tests │       276 |             "An error occurred while trying to attest from Ethereum to Terra"
2023-06-18T14:32:38.6579628Z  sdk-ci-tests │       277 |           );
2023-06-18T14:32:38.6580035Z  sdk-ci-tests │ 
2023-06-18T14:32:38.6580543Z  sdk-ci-tests │       at src/token_bridge/__tests__/terra-integration.ts:274:19
2023-06-18T14:32:38.6581198Z  sdk-ci-tests │           at Generator.throw (<anonymous>)
2023-06-18T14:32:38.6581828Z  sdk-ci-tests │       at rejected (src/token_bridge/__tests__/terra-integration.ts:6:65)
```

**Comment by tbjump** [comment]
Issue occurred again here: https://github.com/wormhole-foundation/wormhole/actions/runs/5361380937/jobs/9727376654

**Comment by evan-gray** [comment]
I expect this has been fixed by #3138, can reopen if not.
